packages/documentation/src/pages/getting-started/workflow/overview.mdx

```diff
@@ -8,15 +8,21 @@ VA.gov uses a [Continuous Integration](https://www.thoughtworks.com/continuous-i
 
 1. [Write](/getting-started/workflow/write)
     - Determine which issue to work on
-    - Create feature branch
+    - Create feature branch (naming convention {issue-id}-{subject you are working on})
     - Code new functionality
     - Write tests for new functionality
+    - Cross browser testing (Chrome, IE, Safari, Edge)
```

Comment:
for the top level items, let's maintain the active voice with the subject (or implied you) first 

`Test new functionality on supported browsers (Chrome, IE, Safari, Edge)`

We should add this to the details page and link to the browsers we support